New: Add billing tab documentation to Console reference (#25)
crate/cloud-reference: docs/overview.rst

.. image:: _assets/img/organization-overview.png
   :alt: Cloud Console organization overview

> The Organization screen consists of five tabs: *Subscriptions*, *Settings*,
> *Users*, the *Audit Log*, and finally *Billing*. By default it shows the
> Subscriptions tab, which displays two things. On the left, it shows the
> possible cloud providers and a link for creating a subscription with either. On
> the right, you see your current subscription (if any) with details: the status
> of the subscription, the subscription ID, the subscription cloud provider, the
> subscription type, and finally, any clusters deployed within the subscription
> with their respective subscription plans.
> 
> The *Deploy cluster* button at the bottom left of any subscription box allows
> you to directly deploy clusters within that subscription. The button with three
> dots at the bottom right allows you to cancel that subscription.
> 
> You can access the other tabs by clicking their respective names at the top.

Reviewer: ```suggestion
The Organization screen consists of five tabs: *Subscriptions*, *Settings*,
*Users*, the *Audit Log*, and finally *Billing*. The Subscriptions tab is default,
which shows possible cloud providers and a link for creating a subscription
with them. You will also details of your current subscription (if any): the
status of the subscription, the subscription ID, the subscription cloud
provider, the subscription type, and finally, any clusters deployed within the
subscription with their respective subscription plans.

The *Deploy cluster* button at the bottom left of any subscription box allows
you to directly deploy clusters within that subscription. The button with three
dots at the bottom right allows you to cancel that subscription.

You can access the other tabs by clicking their respective names at the top.
```

organization's projects, and other such data, each provided with a timestamp.

Organization Billing
--------------------

> This tab shows the billing overview. If you have signed up through the
> recommended route, using a credit card to pay for your cluster, the overview
> of your chosen means of payment will be shown here.
> 
> .. image:: _assets/img/organization-billing.png
>    :alt: Cloud Console organization billing tab
> 
> In case you use multiple cards, a default card can be set. Cards can be deleted
> from the list by using the dots icon on the far right of the card listing.
> Click the *Add payment method* button at the top right to add a new card.

Reviewer: ```suggestion
012345678901234567890123456789012345678901234567890123456789
This tab shows the billing overview. If you signed up with a credit card for your 
cluster (the recommended route), a payment method overview will be shown
here.

.. image:: _assets/img/organization-billing.png
   :alt: Cloud Console organization billing tab

In case you use multiple cards, a default card can be set. Cards can be deleted
from the list by using the dots icon to the right of the card listing.
Click the *Add payment method* button at the top right to add a new card.
```

Author: 012345678901234567890123456789012345678901234567890123456789? 🧐🤷‍♂️

Reviewer: oh darnit, sorry, I meant to remove that ... it's a line length guide.

Reviewer: i removed it now :)

Author: Ah I figured, haha. No probs
